doc/man/m0d.1.txt

```diff
@@ -21,24 +21,172 @@ service as a daemon process in user space.
 
 OPTIONS
 -------
-'TODO: add all options'
+
+Queries:
 
 *-x*::
     List supported transports.
 
 *-l*::
     List supported services.
 
-*-Q*::
-    Minimum TM Receive queue length.
+*-v*::
+    Show version information and exit.
 
 *-h, --help*::
     Display help screen and exit.
 
+Global options:
+
+*-F*::
+    Force mkfs to override found filesystem.
+
+*-Q* num::
+    Minimum length of TM receive queue.
+
+*-M* num::
+    Maximum RPC message size.
+
+*-H* addr::
+    Endpoint address of HA service.
```

Comment:
Please sort these in alphabet order.

Or the man page will sort them automatically?

Reply:
Done. Sorted and updated.